README.md

```diff
@@ -46,7 +46,7 @@ import (
 	"log"
 	"time"
 
-	"github.com/mailru/go-clickhouse"
+	_ "github.com/mailru/go-clickhouse"
```

Comment:
the Array and Date are used from package clickhouse in line 90,91.
it is reason why clickhouse is imported by name.